heal-commands.go

```diff
@@ -296,6 +296,17 @@ func (adm *AdminClient) Heal(ctx context.Context, bucket, prefix string,
 	return healStart, healTaskStatus, nil
 }
 
+// MRFStatus exposes MRF metrics of a server
+type MRFStatus struct {
+	BytesHealed uint64 `json:"bytesHealed"`
+	ItemsHealed uint64 `json:"itemsHealed"`
+
+	TotalItems uint64 `json:"totalItems"`
+	TotalBytes uint64 `json:"totalBytes"`
+
+	Started time.Time `json:"started"`
```

Comment:
what does this signify?  - isn't it always Started?

Reply:
> what does this signify? - isn't it always Started?

Well, we need to show users that there is no MRF activity at a given time. Basically when the MRF queue is empty, then Started will be time sentinel.

So yes, the server will be able to show that there is no MRF activity (basically it means MRF queue for objects is empty)